qemu/qemu.go

```diff
@@ -2739,6 +2768,13 @@ func (config *Config) appendBios() {
 	}
 }
 
+func (config *Config) appendOVMF() {
+	if config.Bios != "" {
+		config.qemuParams = append(config.qemuParams, "-drive")
+		config.qemuParams = append(config.qemuParams, fmt.Sprintf("if=pflash,format=raw,readonly=on,file=%s,", config.Bios))
```

Comment:
Instead of hard-coding its properties, I'd prefer to handle this as a structure that overwrites `Valid()` and `QemuParams()`

```go
type OVMF struct {
    Interface string
    format string
    readonly bool
    file string
}

func (o OVFM) Valid() bool {
...
}

func (o OVFM) QemuParams(config *Config) []string {
...
}

```


what do you think ?

Reply:
Yes, that is better